### PR TITLE
WRO-12044: Increase code coverage for Popup/PopupTabLayout

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Code coverage for Popup and PopupTabLayout needs to be over 80%.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Increased code coverage to 80.88% for Popup and to 61.90% for PopupTabLayout.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
WRO-12044

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: PopupTabLayout/tests/PopupTabLayout-specs.js [hunk 1]
```diff
@@ -72,4 +73,140 @@ describe('PopupTabLayout specs', () => {
...
+						</TabPanels>
+					</Tab>
+				</PopupTabLayout>
+			</FloatingLayerController>
+		);
+
+		const firstTab = screen.getByText('First Tab Title');
+
+		fireEvent.keyDown(firstTab, {keyCode: 13});
+		fireEvent.keyUp(firstTab, {keyCode: 13});
```
Comment (jeonghee27) [inline PopupTabLayout/tests/PopupTabLayout-specs.js:146]: I've already used this way many times, but it's hard to know what key number 13 means.
I think the Slider-spec is more readable.

```

const keyDown = (keyCode) => (elm) => fireEvent.keyDown(elm, {keyCode});

const leftKeyDown = keyDown(37);
const rightKeyDown = keyDown(39);
const upKeyDown = keyDown(38);
const downKeyDown = keyDown(40);

```

CC: @daniel-stoian-lgp @adrian-cocoara-lgp @alexandrumorariu @andreitirlalge

**Comment by juwonjeong** [comment]
Could you please add one more tc to PopupTabLayout to verify panel navigating via left key? I think it can be implemented in a similar way as below.
```
test('should fire onBack event when left key pressed from the second TabPanel', () => {
		const onBack = jest.fn();
		render(
			<FloatingLayerController>
				<PopupTabLayout open>
					<Tab title="Tab Title">
						<TabPanels rtl={false} index={1} onBack={onBack}>
							<TabPanel>
								<Item>Item 1</Item>
								<Item>Item 2</Item>
							</TabPanel>
							<TabPanel>
								<Item>Item 3</Item>
								<Item>Item 4</Item>
							</TabPanel>
						</TabPanels>
					</Tab>
				</PopupTabLayout>
			</FloatingLayerController>
		);

		const firstItem = screen.getByText('Item 3');
		leftKeyDown(firstItem);
		expect(onBack).toHaveBeenCalled();
	});
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**